duckduckgo/iOS
# use DDG's favicon service instead of trying to pull from the web page

Task/Issue URL: https://app.asana.com/0/414235014887631/716678893422644
Tech Design URL:
CC:

**Description**:

Use DDG's favicon service rather than trying to get them via Javascript

**Steps to test this PR**:

1. In a previous version of the app, create some book marks.
1. Open this version of the app, check existing bookmarks have the correct icons.
1. Add a new bookmark, check the icon is correct
1. Open and close some tabs, check the icon in tab switcher is correct


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
